x-pack/plugins/observability_solution/inventory/server/routes/entities/get_entity_types.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { type ObservabilityElasticsearchClient } from '@kbn/observability-utils/es/client/create_observability_es_client';
+import { ENTITIES_LATEST_ALIAS, EntityType } from '../../../common/entities';
+import { ENTITY_DEFINITION_ID, ENTITY_TYPE } from '../../../common/es_fields/entities';
+import {
+  BUILTIN_CONTAINERS_FROM_ECS_DATA,
+  BUILTIN_HOSTS_FROM_ECS_DATA,
+  BUILTIN_SERVICES_FROM_ECS_DATA,
+  DEFAULT_ENTITY_TYPES,
+} from './get_latest_entities';
+
+export async function getEntityTypes({
+  inventoryEsClient,
+}: {
+  inventoryEsClient: ObservabilityElasticsearchClient;
+}) {
+  const entityTypesEsqlResponse = await inventoryEsClient.esql('get_entity_types', {
+    query: `FROM ${ENTITIES_LATEST_ALIAS}
+    | WHERE ${ENTITY_TYPE} IN (${DEFAULT_ENTITY_TYPES.map(
+      (entityType) => `"${entityType}"`
+    ).join()}) 
+     | WHERE ${ENTITY_DEFINITION_ID} IN (${[
```

Comment:
we already use this clause in two places and most likely we'll use it in more. 
Do you think we can abstract this clause somehow so we can reuse it?  same fr the entity_type.

Reply:
yeah, I knew you'd comment on this. I'll create a helper and have it shared, we only have a few queries now.

Reply:
thanks 🙏